test/e2e/set/scyllacluster/scyllacluster_upgrades.go

```diff
@@ -37,9 +37,28 @@ var _ = g.Describe("ScyllaCluster upgrades", func() {
 			ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
 			defer cancel()
 
-			sc := scyllafixture.BasicScyllaCluster.ReadOrFail()
+			sc := f.GetDefaultScyllaCluster()
 			sc.Spec.Version = e.initialVersion
 
+			supportsExposing, err := scyllafeatures.Supports(sc, scyllafeatures.ExposingScyllaClusterViaServiceOtherThanClusterIP)
```

Comment:
why does this relates to upgrades in particular?

Reply:
We only have one minor version of Scylla OS which is capable of exposing nodes on PodIPs, so we can't run the upgrade test with PodIPs, and have to modify the default cluster accordingly (which I assume is better than skipping the test).